src/drivers/mct/bld/namelist_files/namelist_defaults_drv.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+
+<?xml-stylesheet type="text/xsl" href="namelist_defaults.xsl"?>
+
+<namelist_defaults>
+
+<drv_flds_in_files>Buildconf/camconf/drv_flds_in,Buildconf/clmconf/drv_flds_in</drv_flds_in_files>
+
+</namelist_defaults>
```

Comment:
I don't believe ALM is using this file. The [d0be438](https://github.com/ACME-Climate/ACME/commit/d0be438b6b3822d8456c7512d7cf86bc2174339a) in ACME by @agsalin moved `cime/driver_cpl/bld/namelist_files/namelist_defaults_drv.xml` to `src/drivers/mct/bld/namelist_files/namelist_defaults_drv.xml`.